api/spec/smithy/model/operations/GetClusterLogEvents.smithy

```diff
@@ -0,0 +1,43 @@
+namespace parallelcluster
+
+@suppress(["MissingPaginatedTrait", "PaginatedTrait", "InputOutputStructureReuse"])
+@paginated
+@readonly
+@http(method: "GET", uri: "/v3/clusters/{clusterName}/logstreams/{logStreamName}", code: 200)
+@tags(["Cluster Logs"])
+@documentation("Retrieve the events associated with a log stream.")
+operation GetClusterLogEvents {
+    input: GetClusterLogEventsRequest,
+    output: GetLogEventsResponse,
+    errors: [
```

Comment:
we probably also need NotFoundException and in the message we should discriminate between cluster not found and log stream not found

same for image